docker/Dockerfile.geth

```diff
@@ -1,8 +1,25 @@
-FROM ethereum/client-go
+# Install Ethereum client (geth) from Docker image
+FROM ethereum/client-go AS geth
 
-EXPOSE 8545
+# Base image for running Node.js applications
+FROM node:18-bullseye
+
+# Set the working directory
+WORKDIR /zkevm-contracts
+
+# Copy package.json and package-lock.json to the container
+COPY package*.json ./
+
+# Install zkevm-contracts dependencies
+RUN npm install
```

Comment:
since the `zkevm-contracts` repo is only supposed to run once to deploy the initial contracts, I think we probably don't need to include the source code in the docker image which is supposed to start a geth client 

so instead, I think we should create a new docker image that contains the `zkevm-contracts` source code and is supposed to execute a one-time command in another container that runs pure geth and exit, similar to the `zkevm-approve` container.